### ISSUE TITLE
Lint regex

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#688]
# Rule request

## Thesis

We need to find existing plugin to lint regex in python.
Currently it might contain issues.

## Reasoning

Regex is the important part of python development. And we do not lint it anyhow.

**Comment by sobolevn** [comment]
Out of scope. And too complex. We even can't tell what string is regex and what is not.

**Comment by sobolevn** [comment]
Related: https://github.com/PyCQA/bandit/issues/589

**Comment by mschwager** [comment]
Dlint's rule [`DUO138`](https://github.com/dlint-py/dlint/blob/master/docs/linters/DUO138.md) will detect catastrophic regular expressions. There are still some false positives, but it's right for the most part. Since Dlint is built on `flake8` you can simply add `dlint` to your dependencies and it should work.

If you only want `DUO138` for now, then you can disable all Dlint rules and only enable `DUO138`. I think the following in `setup.cfg` should do the trick:

```
ignore = D100, D104, D401, W504, RST303, RST304, DAR103, DAR203, DUO
select = E, F, W, C90, DUO138
```

This is the default selected rules + `DUO138`.
